sources/install/package_ad.sh

```diff
@@ -111,8 +112,8 @@ function install_bloodhound-py() {
     colorecho "Installing and Python ingestor for BloodHound"
     pipx install git+https://github.com/fox-it/BloodHound.py
     # https://github.com/dirkjanm/BloodHound.py/pull/146
-    local TEMP_FIX_LIMIT="2024-02-15"
-    if [ "$(date +%Y%m%d)" -gt "$(date -d $TEMP_FIX_LIMIT +%Y%m%d)" ]; then
+    local temp_fix_limit="2024-02-15"
```

Comment:
fyi, this needs to be changed as well in the doc ([here](https://exegol.readthedocs.io/en/latest/community/contributors.html#temporary-fixing-a-tool)). I did the change now

Reply:
Thanks